Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG UBUNTU_VER=16.04
+FROM ubuntu:${UBUNTU_VER}
+
+LABEL maintainers="mihai.cojan@centrys.io, alexandru.laurus@centrys.io"
+
+# prepare for java instalation
+RUN apt-get update && apt-get install -y bzip2 lsb-release wget curl jq locales net-tools libicu-dev libedit-dev
+RUN apt-get clean
+
+# change locales to UTF-8 in order to avoid bug when changing config.xml
+RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && locale-gen
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+WORKDIR /opt
+
+# TODO: use java 10.0.2
+ARG DEV_BUILD=false
+RUN if [ "${DEV_BUILD}" = "true" ]; then wget -O jdk.tar.gz \
+https://download.java.net/java/GA/jdk10/10.0.1/fb4372174a714e6b8c52526dc134031e/10/openjdk-10.0.1_linux-x64_bin.tar.gz; fi
```

Comment:
@aion-kelvin This is just for local development/remote debugging as the kernel binary packs it's own version of Oracle Java which are missing the debug libs (`libdt_socket.so` and `libjdwp.so`).

The initial approach was to use an official java image, however there are a few reasons which made me choose the hardcoded URL way:
* The kernel binary comes with it's own version which I understand is prone to changes. Having the change lower in the Dockerfile declaration, instead of first line for base image will preserve space as upper layers can be reused.
* There are more (system) dependencies required to be installed with the java official image
* This is just for development, official (release) images won't need this/wont' be affected by it

I'll update the `README.md` with a better description of what is/is not development.